graphql_compiler/query_formatting/common.py

```diff
@@ -21,8 +21,20 @@
 # Public API
 ######
 
+def _raise_invalid_type_error(expected_python_type_name, value, name):
+    """Raise a GraphQLInvalidArgumentError that states that the argument type is invalid."""
```

Comment:
Please write a full docstring here. It's far from obvious that `name` could be `None`.